guide/voice/voice-broadcasts.md

```diff
@@ -1,3 +1,105 @@
+:::
+
+## Playing audio
+
+Playing audio on a VoiceBroadcast is virtually the same as if you tried to play audio on a VoiceConnection (the only caveat being that you cannot play a VoiceBroadcast onto another VoiceBroadcast).
+
+```js
+const fs = require('fs');
+
+const broadcast = client.voice.createBroadcast();
+
+// From a path
+broadcast.play('audio.mp3');
+// From a ReadableStream
+broadcast.play(fs.createReadStream('audio.mp3'));
+// From a URL
+broadcast.play('http://myserver.com/audio.aac');
+```
+
+The `.play()` method on a VoiceBroadcast returns a `BroadcastDispatcher` that can be used to control the playback of the audio stream like a regular `StreamDispatcher`.
+
+Playing a broadcast on a voice connection (i.e. to subscribing to the broadcast) is really easy!
+
+```js
+// All 3 connections will play the same broadcast - they will play audio at the same time
+connection1.play(broadcast);
+connection2.play(broadcast);
+connection3.play(broadcast);
+```
+
+## Key takeaways
+
+There are some things to be aware of with broadcasts, this section will help you to understand them.
+
+### VoiceBroadcast
+- Can be created with `client.voice.createBroadcast()` when you're going to be playing the same audio across more than one `VoiceConnection`.
+- Playing the same broadcast across multiple connections offers much better performance and synchronicity compared to playing the same resource across each connection individually - much of the intensive audio processing is done only once.
+- `subscribe` and `unsubscribe` events are emitted here when VoiceConnections are subscribed and unsubscribed.
+	```js
+	broadcast.on('subscribe', dispatcher => {
+		console.log(`Broadcast playing in ${dispatcher.player.voiceConnection.channel.name}`);
+	});
+
+	// Emits the subscribe event on the broadcast
+	connection.play(broadcast);
+	```
+- You can use the `.play(resource)` method to play audio on the broadcast and across all subscribers.
+- You can use the `.end()` method to destroy the broadcast, ending playback across all subcribers and unsubscribing them. This will also remove the broadcast from the `client.voice.broadcasts` array.
+	```js
+	// Destroy a broadcast
+	broadcast.end();
+	```
+
+### BroadcastDispatcher
+```js
+const broadcastDispatcher = broadcast.play('music.mp3');
+```
+- Subscribers are `VoiceConnection`s that are currently playing the broadcast.
+- A central controller for all subscribers - any changes to this dispatcher will affect all subscribers.
+- Pausing/resuming this dispatcher will pause/resume playback across all subscribers.
+- Altering the volume of this dispatcher will alter the volume across all subscribers.
+- Destroying this dispatcher will stop playback across all subscribers, but they will still remained to the broadcast as the broadcast itself has not been destroyed. To end the broadcast entirely and unsubscribe all subscribers, you should use `broadcast.end()`.
```

Comment:
The bit beyond the comma seems jumbled. Maybe you forgot to delete "... ed to the broadcast", so that it reads "but they will still remain as the broadcast itself..."?